ouroboros-consensus/docs/tutorials/Ouroboros/Consensus/Tutorial/Simple.lhs

```diff
@@ -542,12 +544,11 @@ Again, the slot abstraction defines a logical clock - and instances of the
 As such, we will also need to define an instance of `Ticked` for our ledger
 state.  In our example, this is essentially an `Identity` functor:
 
-> newtype instance Ticked (LedgerState BlockC) =
+> newtype instance Ticked1 (LedgerState BlockC) mk =
```

Comment:
The section header here mentions `Ticked` while the data family instance is for `Ticked1`